<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mri_proj_Widget</class>
 <widget class="QWidget" name="mri_proj_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proj</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="plotWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="plotLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="allProjWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>850</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="allProjLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controlWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>850</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <widget class="QComboBox" name="projAxisValue">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>270</y>
           <width>90</width>
           <height>26</height>
          </rect>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="shimLabel">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>530</y>
           <width>120</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Shim strength</string>
         </property>
        </widget>
        <widget class="QPushButton" name="acquireButton">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>320</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Acquire</string>
         </property>
        </widget>
        <widget class="QPushButton" name="startButton">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>150</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QLabel" name="shimLabel_y">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>530</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
        <widget class="QLabel" name="freqLabel">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>90</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Frequency/MHz</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="shim_z">
         <property name="geometry">
          <rect>
           <x>305</x>
           <y>560</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="axisLabel">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>270</y>
           <width>150</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Projection Axis</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="shim_x">
         <property name="geometry">
          <rect>
           <x>185</x>
           <y>560</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="stopButton">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>150</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
        <widget class="QLabel" name="shimLabel_x">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>530</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
        <widget class="QPushButton" name="loadShimButton">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>610</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>load shims</string>
         </property>
        </widget>
        <widget class="QPushButton" name="zeroShimButton">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>660</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>zero shims</string>
         </property>
        </widget>
        <widget class="QLabel" name="ProjLabel">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>221</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Projection control panel</string>
         </property>
        </widget>
        <widget class="QLabel" name="shimUnitLabel">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>560</y>
           <width>30</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>mA</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="shim_y">
         <property name="geometry">
          <rect>
           <x>245</x>
           <y>560</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="shimLabel_z">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>530</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="freqValue">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>90</y>
           <width>95</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="maximum">
          <double>60.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>15.670000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="shimmingLabel">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>500</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Shimming tools</string>
         </property>
        </widget>
        <widget class="QPushButton" name="acquireAllButton">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>370</y>
           <width>210</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Get all projections</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
